core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -261,19 +267,16 @@ public void addLostStorage(Map<String, StorageList> lostStorage) {
   /**
    * Gets the selected field information for this worker.
    *
-   * You should lock externally with {@link MasterWorkerInfo#lockWorkerMeta(EnumSet, boolean)}
-   * with {@link WorkerMetaLockSection#USAGE} specified.
-   * A shared lock is required.
+   * You should lock externally with {@link MasterWorkerInfo#lockWorkerMetaForInfo(Set)}.
+   * The required locks will be determined internally based on the fields.
    *
    * @param fieldRange the client selected fields
    * @param isLiveWorker the worker is live or not
    * @return generated worker information
    */
   public WorkerInfo generateWorkerInfo(Set<WorkerInfoField> fieldRange, boolean isLiveWorker) {
     WorkerInfo info = new WorkerInfo();
-    Set<WorkerInfoField> checkedFieldRange = fieldRange != null ? fieldRange :
-        new HashSet<>(Arrays.asList(WorkerInfoField.values()));
```

Comment:
We used to allow `null` and handle here. `null` is no longer allowed, in order to avoid confusion.